railties/test/generators/plugin_generator_test.rb

```diff
@@ -148,6 +148,32 @@ def test_generating_adds_dummy_app_without_sprockets
     end
   end
 
+  def test_generator_for_yarn
```

Comment:
The test `test_generator_for_yarn` in `railties/test/generators/app_generator_test.rb` different from `test_generator_for_yarn` in `railties/test/generators/plugin_generator_test.rb`

Expample:
```ruby
assert_file "package.json", /dependencies/ #  railties/test/generators/app_generator_test.rb
assert_file "test/dummy/package.json", /dependencies/ # railties/test/generators/plugin_generator_test.rb
```